app/jobs/notifications_worker.rb

```diff
@@ -1,12 +1,10 @@
 class NotificationsWorker < ActiveJob::Base
   queue_as :default
 
-  def perform(contact_info, message)
-    phone_number = contact_info[:phone_number]
+  def perform(contact_info, message, type)
+    phone_number = contact_info[:phone_number].to_s
     address = contact_info[:email_address]
-    response = SmsService.new.send_sms(phone_number, message) if phone_number
-    if !phone_number || response['response'] != 'success'
-      AppointmentNotif.initial_message(address, message).deliver_now
-    end
+    sms = SmsService.new.send_sms(phone_number, message)
```

Comment:
Matt, does this still work without the check for the phone number to be present?

Reply:
Yes, it does work. I made some changes to the microservice.